docs/tutorials/projects/tipsy.md

```diff
@@ -664,28 +664,28 @@ When we write the code for the robot, we can differentiate between, say, a perso
 
 ## Design your robot
 
-{{<imgproc src="/tutorials/tipsy/assembly-frame.jpg" resize="400x" declaredimensions=true alt="four T-slotted framing rails connected to the Scuttle base with brackets for height, and two T-slotted framing rails connected to underneath the box with brackets for holding the drinks." class="alignleft">}}
+{{<imgproc src="/tutorials/tipsy/assembly-frame.jpg" resize="400x" declaredimensions=true alt="four T-slotted framing rails connected to the SCUTTLE base with brackets for height, and two T-slotted framing rails connected to underneath the box with brackets for holding the drinks." class="alignleft">}}
 
 Now that you have all your components wired, configured, and tested, you can assemble your robot.
 
-Add four 3’ long T-slotted framing rails along the corners of the Scuttle Rover base to make it a tall structure.
+Add four 3’ long T-slotted framing rails along the corners of the SCUTTLE Rover base to make it a tall structure.
 Then add two 12 inches long T-slotted framing rails in the middle of the structure at the height that you want to hold the box.
 Secure them using T-slotted framing structural brackets.
 
 Next, add the wired Raspberry Pi, motor driver, and 12V battery to the base.
 
-You can use the 3D-printed holders that come with the assembled Scuttle base for the Raspberry Pi and the motor driver.
-You can also print holders based on Scuttle designs from [grabcad](https://grabcad.com/library?page=1&time=all_time&sort=recent&query=scuttle).
+You can use the 3D-printed holders that come with the assembled SCUTTLE base for the Raspberry Pi and the motor driver.
+You can also print holders based on SCUTTLE designs from [GrabCAD](https://grabcad.com/library?page=1&time=all_time&sort=recent&query=scuttle).
 
-Secure the 12V battery to the bottom using velcro tape or other tape, and secure the sides using T-slotted brackets.
+Secure the 12V battery to the bottom using Velcro tape or other tape, and secure the sides using T-slotted brackets.
 
 {{<imgproc src="/tutorials/tipsy/assembly-rails-velcro.jpg" resize="500x" declaredimensions=true alt="velcro tape on T-slotted rails secured from the sides." class="aligncenter">}}
 
-Secure the buck converter with velcro tape, double-sided tape, or a 3D printed holder.
+Secure the buck converter with hook-and-loop tape, double-sided tape, or a 3D printed holder.
```

Comment:
so hook-and-loop or hook and loop ?

Reply:
unclear. I've seen it both ways. I guess we can go hyphenated for consistency but there's also https://www.hookandloop.com/blog/theres-no-such-thing-as-a-velcro-strap